### ISSUE TITLE
[3.3.5] DB/Spell: Roc Form (35200)

### ISSUE THREAD

**Issue by Treeston** [TrinityCore/TrinityCore#16164]
Branch 3.3.5, rev. 8a0bbc386982
[Spell: Roc Form](http://www.wowhead.com/spell=35200/roc-form)
[Creature: Feralfen Druid](http://www.wowhead.com/npc=20270/feralfen-druid)
Visual for transformation should be a Roc (duh). It is a bear on current TC.

**Comment by Saben65** [comment]
Not sure if this helps at all.
```
ServerToClient: SMSG_AURA_UPDATE (0x10AF) Length: 54 ConnIdx: 1 Time: 01/03/2016 20:59:42.750 Number: 299
UpdateAll: False
UnitGUID: Full: 0x1C0194424013CB800000270000098980 Creature/0 R101/S39 Map: 530 Entry: 20270 Low: 625024
AurasCount: 1
[0] Slot: 0
[0] HasAura: True
[0] SpellID: 35200 (35200)
[0] SpellXSpellVisualID: 6582
[0] Flags: NoCaster, Positive, Duration (7)
[0] ActiveFlags: 1
[0] CastLevel: 62
[0] Applications: 0
[0] Int56 Count: 0
[0] Effect Count: 0
[0] HasCastUnit: False
[0] HasDuration: True
[0] HasRemaining: True
[0] Duration: 300000
[0] Remaining: 300000
```

**Comment by tkrokli** [comment]
Looks interesting to me, since it has `SpellID` and `SpellVisualID`. I am not a real coder, though. ;)

**Comment by Killyana** [comment]
It's weird: Apply Aura: Shapeshift (Creature - Bear)
Maybe there's a manual morph that must be done, but I cannot find the model_ID

**Comment by tkrokli** [comment]
From the wowhead description: "Shapeshifts into a roc for 5 min., increasing armor and hit points, as well as allowing the use of various bear abilities." What about that sniff info SpellXSpellVisualID: 6582 , CastFlags: 768 or CastFlagsEx: 16 , doesn't that help anything?

**Comment by Saben65** [comment]
Based on a visual comparison alone, http://www.wowhead.com/npc=22265/shadowwing-owl seems to have the same model and they are from the same expansion which makes it seem more plausible. The model ID would then be 10831.

![owl](http://i65.tinypic.com/206d09i.png)

**Comment by Aokromes** [comment]
maybe fixed by https://github.com/TrinityCore/TrinityCore/commit/6ba0d489adbe2e93d3c9060944bb4c6f0d3d0259 ?

**Comment by Jildor** [comment]
>maybe fixed by 6ba0d48 ?

@Aokromes I think it is a effect shapeshift, not effect transform
